Graphs/Table/index.js

```diff
@@ -185,31 +226,32 @@ class Table extends AbstractGraph {
         return configuration.data.columns;
     }
 
-    getAccessor(columns) {
-        return columns.map(columnAccessor);
-    }
+    getKeyColumns() {
+        const {
+            data
+        } = this.props;
 
-    getTooltipAccessor(columns) {
-        return columns.map(column => {
-            return column.tooltip ? columnAccessor(column.tooltip) : () => {}
-        });
+        if (!data || !data.length)
+            return;
```

Comment:
return an empty array as you will get a stack trace in the case below when there is no data